src/Action.hs

```diff
@@ -122,6 +123,7 @@ module Action
     , module Action.Smtp
     , sendMailToRole
 
+    , module Logger
```

Comment:
just for the record: i also dislike re-exports.  they make it hard to get oriented, and i don't see a great benefit in using them.